src/ui/components/BoxScore.hockey.tsx

```diff
@@ -42,20 +43,32 @@ const StatsTable = ({
 
 	const [sortBys, setSortBys] = useState(() => {
 		return PLAYER_GAME_STATS[type].sortBy.map(
-			stat => [stats.indexOf(stat), "desc"] as SortBy,
+			stat => [`stat:${stat}`, "desc"] as SortBy,
 		);
 	});
 
-	const onClick = (event: MouseEvent, i: number) => {
-		setSortBys(
-			prevSortBys =>
+	const onClick = (event: MouseEvent, colKey: string) => {
+		setSortBys(prevSortBys => {
+			const newSortBys =
 				updateSortBys({
 					cols,
 					event,
-					i,
+					colKey,
 					prevSortBys,
-				}) ?? [],
-		);
+				}) ?? [];
+
+			if (
+				newSortBys.length === 1 &&
+				prevSortBys.length === 1 &&
+				newSortBys[0][0] === prevSortBys[0][0] &&
+				newSortBys[0][1] === "desc"
+			) {
+				// User just clicked twice on the same column. Reset sort.
```

Comment:
Why add this here? It's in basketball because the default sort order (starters first) is not possible by just sorting a visible column. But that's not true for hockey. Same question for football.

Reply:
after merging i was having an issue with the box scores, so i fixed it for basketball and copy pasted it for hockey and football. will remove

Reply:
removed the double click on hockey and football